### PR TITLE
CEDS-1475 Fix personal ucr bug

### PR DESCRIPTION
**Pull request by PauloPortugal**
* Align PersonalUCR name with Backend
 * Change DUCR regex validation
 * Review content and validation on PreviousDocuments page

### CODE REVIEW

File: app/forms/Ducr.scala [hunk 1]
```diff
@@ -26,7 +26,11 @@ case class Ducr(ducr: String)
 object Ducr {
   implicit val format = Json.format[Ducr]
 
-  private val ducrFormat = "^\\d[A-Z]{2}\\d{12}-[0-9A-Z]{1,19}$"
+  private val ducrFormat = """^[0-9][A-Z][A-Z][0-9A-Z\(\)\-/]{6,32}|
```
Comment (bambuchaAdm) [inline app/forms/Ducr.scala]: Where is specification for this regexp ?

Comment (PauloPortugal) [reply app/forms/Ducr.scala]: Is in a tariff document.
They accept 5 different DUCR formats.
I am just getting this confirmed with Ian.

We have used the same for the movements project.

Comment (bambuchaAdm) [reply app/forms/Ducr.scala]: Would you like to put this as scaladoc over - just to not lose connection from where it comes.

Comment (PauloPortugal) [reply app/forms/Ducr.scala]: This has been addressed.

**Pull request merged**